org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/PluginImageHelper.java

```diff
@@ -65,6 +66,8 @@
 	@Named("org.eclipse.xtext.ui.PluginImageHelper.notFound")
 	private String notFound = "notFound.gif"; //$NON-NLS-1$
 
+	private DisplayDisposeListener displayDisposeListener = null;
```

Comment:
Since the field is private, the static inner class doesn't need  to be public I suppose.